wgsl/index.bs

```diff
@@ -12303,13 +12455,13 @@ the rules in [[#floating-point-overflow]] apply.
     The infinitely precise result is computed as either `min(max(x,low),high)`, or with a median-of-3-values formulation.
     These may differ when `low > high`.
 
-    If `x` and either `low` or `high` are denormalized, the result may be any of the denormalized values.
-    This follows from the possible results from the `min` and `max` functions on denormalized inputs.
+    If `x` and either `low` or `high` are [=ieee754/subnormal=], the result may be any of the [=ieee754/subnormal=] values.
+    This follows from the possible results from the `min` and `max` functions on [=ieee754/subnormal=] inputs.
   <tr><td>`cos(x)`
       <td>Absolute error at most 2<sup>-11</sup> when `x` is in the interval [-&pi;, &pi;]
       <td>Absolute error at most 2<sup>-7</sup> when `x` is in the interval [-&pi;, &pi;]
   <tr><td>`cosh(x)`<td colspan=2 style="text-align:left;">Inherited from `(exp(x) + exp(-x)) * 0.5`
-  <tr><td>`cross(x, y)`<td colspan=2 style="text-align:left;">Inherited from `(x[i] * y[j] - x[j] * y[i])`
+  <tr><td>`cross(a, b)`<td colspan=2 style="text-align:left;">Inherited from `(a[i] * b[j] - a[j] * b[i])` where `i` &ne; `j`
```

Comment:
Why the switch to `a` and `b` here? Almost everything else in this table uses `x`, `y`, and `z` for for parameters.

Reply:
because I switched the notation in definition of the function. Will switch back.

This one actually threw me for a loop because the formula is a lot more complicated than hinted at here.

Will switch back to x and y.